apps/examples/webserver_test/readme.md

```diff
@@ -0,0 +1,45 @@
+# Webserver Test Application
+
+### Features
+
+- HTTPS (based on mbedTLS h/w accelerate)
```

Comment:
Test script supports both HTTP / HTTPS protocol.

is it really required, as test will work for both H/W and S/W TLS operation